### PR TITLE
Errors need to be fixed

### PR DESCRIPTION
**Pull request by hs115488**
I see a lot of broken  and empty files. You have to stop uploading files using the website. You're breaking the code every time you do that. You have to use Github Desktop. Your files are broken or empty. 

You also have code that are copied and pasted. That's disappointing. That's shows you don't know what you are doing. Please fix that

### CODE REVIEW

File: Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb [hunk 1]
```diff
@@ -55,188 +67,26 @@
...
-       "1257  60154200  \n",
-       "1258  33354500  \n",
-       "\n",
-       "[1259 rows x 7 columns]"
-      ]
-     },
-     "execution_count": 3,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "FileNotFoundError",
```
Comment (hs115488) [inline Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb:74]: Missing a file dependency

File: Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb [hunk 2]
```diff
@@ -36,9 +36,21 @@
...
   {
    "cell_type": "code",
-   "execution_count": 33,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'fbprophet'",
```
Comment (hs115488) [inline Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb:44]: You don't have a virtual environment set up for others so the file will always be broken to others

File: Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb [hunk 3]
```diff
@@ -253,20 +103,19 @@
...
-      "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
+     "ename": "NameError",
+     "evalue": "name 'aapl' is not defined",
```
Comment (hs115488) [inline Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb:111]: Missing file

File: Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb [hunk 4]
```diff
@@ -292,197 +141,40 @@
...
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'fbprophet' is not defined",
```
Comment (hs115488) [inline Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb:149]: Missing virtual environment. Means every time someone downloads the file, it's broken

File: Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb [hunk 5]
```diff
@@ -292,197 +141,40 @@
...
-       "1258  33354500  \n",
-       "\n",
-       "[1259 rows x 7 columns]"
-      ]
-     },
-     "execution_count": 12,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "NameError",
+     "evalue": "name 'aapl' is not defined",
```
Comment (hs115488) [inline Quantitative Analysis/Apple (AAPL) Quantitative Analysis  (2).ipynb:170]: Broken dependency

**Comment by kelbahloul** [comment]
Deleted Quantitative Analysis folder and web app folder, Apple analysis is going to be ongoing

**Pull request closed without merging**